### PR TITLE
[PHPTests:Team] Add Team Controller Tests 

### PR DESCRIPTION
**Pull request by shailpatels**
My first test!
Here's an example of starting to create a new test!

This PR updates the createTeam function to use the submitty JSON response functions
and adds tests for it.

### CODE REVIEW

**Comment by zjxiaohan** [comment]
Hi, could the title be made more meaningful?

**Comment by shailpatels** [comment]
> Hi, could the title be made more meaningful?

This PR is more of an example right now for people to start writing tests, I don't expect it to be merged right now :)

In the future if this branch does get worked on then yes I'll make it more descriptive!

File: site/tests/app/controllers/student/TeamControllerTester.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+		$controller = new TeamController($this->core);
+		//build folders for new team
+		$this->assertTrue(FileUtils::createDir($this->config['gradeable_path'], null, true));
+		$this->core->getQueries()->method('createTeam')->willReturn("test");
+		$tmp = FileUtils::joinPaths($this->config['gradeable_path'], "test");
+		$this->assertTrue(FileUtils::createDir($tmp, null, true));
+
+		$response = $controller->run();
+
+		$this->assertEquals(["status" => "success", "data" => null] , $response);
```
Comment (MasterOdin) [inline site/tests/app/controllers/student/TeamControllerTester.php]: what are additional asserts that you can add here to help validate the function did what you wanted it to beyond just that it returned with a success status?

**Comment by bmcutler** [comment]
@shailpatels Please resolve conflicts

**Pull request merged**